gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/TransientMapFeatures.java

```diff
@@ -19,35 +19,42 @@
 import static java8.util.stream.StreamSupport.stream;
 
 import com.google.android.gnd.model.feature.Point;
-import com.google.android.gnd.model.feature.PolygonFeature;
 import com.google.android.gnd.model.layer.Style;
 import com.google.android.gnd.ui.map.MapFeature;
 import com.google.android.gnd.ui.map.MapPin;
 import com.google.android.gnd.ui.map.MapPolygon;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableSet;
 
+/** Helper class to generate ephemeral {@link MapFeature}. Used when adding/editing features. */
```

Comment:
These methods are only used by `MapContainerViewModel`, so should probably live there. In the future we should probably refactor the bits of `MapContainerViewModel` that have to do with polygon drawing into their own VM or helper VM somehow.

Reply:
Done